docs/src/main/asciidoc/security-oidc-auth0-tutorial.adoc

```diff
@@ -0,0 +1,743 @@
+[id="security-oidc-auth0-tutorial"]
+= Protect Quarkus web application by using `Auth0` OpenID Connect provider
+include::_attributes.adoc[]
+:diataxis-type: tutorial
+:categories: security,web
+
+xref:security-architecture.adoc[Quarkus Security] provides comprehensive OpenId Connect (OIDC) and OAuth2 support with its `quarkus-oidc` extension, supporting both xref:security-oidc-code-flow-authentication.adoc[Authorization code flow] and xref:security-oidc-bearer-token-authentication.adoc[Bearer token] authentication mechanisms.
+
+Well-known OIDC providers such as https://www.keycloak.org/documentation[Keycloak], https://developer.okta.com/[Okta], https://auth0.com/docs/[Auth0] and many xref:security-openid-connect-providers.adoc[social providers] can be configured with ease.
```

Comment:
```suggestion
With Quarkus, you can easily configure xref:security-openid-connect-providers.adoc[well-known social OIDC and OAuth2 providers], including link:https://www.keycloak.org/documentation[Keycloak], link:https://developer.okta.com/[Okta], and link:https://auth0.com/docs/[Auth0].
```

Reply:
Hi Michelle, would you be OK if I tried:
```
With Quarkus, you can easily configure OIDC providers such as link:https://www.keycloak.org/documentation[Keycloak], link:https://developer.okta.com/[Okta], and link:https://auth0.com/docs/[Auth0], and xref:security-openid-connect-providers.adoc[well-known social OIDC and OAuth2 providers]
```
?

See, `Keycloak`, `Okta` and `Auth0`, these are not qualified as `social` providers, they are `enterprise` grade (though I would not use `enterprise` here), one thing is certain is that when you see social login options, they'd never offer Keycloak or Okta or Autho :-), while xref:security-openid-connect-providers.adoc (somewhat misnamed file name) lists `social` providers only.

Reply:
@sberyozkin -  Your suggestion is perfect +1. I see what you mean. Thanks for explaining. Apologies for the churn.

Reply:
@michelle-purcell Np at all, thanks for trying to improve the text